### PR TITLE
Tests / move CDN handling to hook

### PR DESCRIPTION
**Pull request by tom-james-watson**

### CODE REVIEW

**Comment by tom-james-watson** [comment]
[Clean up Digital Assets module](https://trello.com/c/LKbhEqvF/342-clean-up-digital-assets-module)

File: src/storage/google-cloud.js [hunk 1]
```diff
@@ -3,8 +3,7 @@ import filenameHandler from './filename'
 import meter from 'stream-meter'
 import multer from 'multer'
 import path from 'path'
-const gcloud = require('google-cloud')
-const storage = gcloud.storage
+const storage = require('@google-cloud/storage');
```
Comment (chrisabrams) [inline src/storage/google-cloud.js]: YESSSS

File: .env.sandbox.example [hunk 2]
```diff
@@ -5,8 +5,11 @@ CHANGE_FEED                 = false
 EXPRESS_PORT                = 8400
 
 GCLOUD_KEYPATH              = './credentials.json'
-GCLOUD_PROJECT              = ''
-GCLOUD_STORAGE_BUCKET       =
+GCLOUD_PROJECT              = f12-tech
+GCLOUD_STORAGE_BUCKET       = f12-general
```
Comment (chrisabrams) [inline .env.sandbox.example]: We don't want to commit these items. It's actually considered a security exposure according to Google Cloud. I'd recommend a force push commit that doesn't include this change. The keypath is fine, that's just a path to a file.

File: test/integration/model.js [hunk 3]
```diff
@@ -1,7 +1,15 @@
-import codes from '../../src/lib/error-codes'
 import DigitalAssetModel from '../../src/models/digital-asset'
+import MockHTTPServer from '../helpers/mock-http-server'
 import Migrations from 'structure-migrations'
 import pluginsList from '../helpers/plugins'
+import TestAPI from '../helpers/test-api'
+import OrgTestAPI from 'structure-organizations/test/helpers/test-api'
+import AppTestAPI from 'structure-applications/test/helpers/test-api'
```
Comment (chrisabrams) [inline test/integration/model.js]: Oh I hadn't realized you had set it up to import some helpers this way, nice!

**Comment by tom-james-watson** [comment]
I didn't realise there were various types of cdn. I'll turn it into its on plugin in that case.

**Pull request merged**